src/containers/CaseDetail/CaseDetail.js

```diff
@@ -7,50 +7,148 @@ import UserDetailedSymptomsCard from "../../components/UserDetailedSymptomsCard/
 import CasesAPI from "../../services/CasesAPI";
 import {useParams} from "react-router";
 import ReportsAPI from "../../services/ReportsAPI";
+import TelemetricAPI from "../../services/TelemetricAPI";
 
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import styled from "styled-components";
 import {useAuth0} from "../../shared/Auth";
+import DoctorUserCommunication from "../DoctorUserCommunication/DoctorUserCommunication";
+import ConductCard from "../../components/ConductCard/ConductCard";
 
 const CaseDetail = () => {
     //TODO: LOAD FROM HISTORY PROPS THE PATH to get the caseId
 
-    const { loading } = useAuth0()
-    const {caseId} = useParams()
+    const {loading, isAuthenticated, token, user} = useAuth0()
+    const {id} = useParams()
     const [caseState, setCaseState] = useState([])
     const [report, setReport] = useState({})
+    const [vitalSigns, setVitalSigns] = useState({});
+    const [lastConduct, setLastConduct] = useState('');
+    const [diagnosisAndConduct, setDiagnosisAndConduct] = useState({});
+    const [diagnosisSaveDisabled, setDiagnosisSaveDisabled] = useState(true);
+    const [conductSaveDisabled, setConductSaveDisabled] = useState(true);
 
     useEffect(() => {
-        const loadCaseState = async (caseId) => {
-            const states = await CasesAPI.getStatesForCase(caseId)
+        const loadCaseState = async (id, token) => {
+            const states = await CasesAPI.getStatesForCase(id, token)
             setCaseState(states)
         }
-        loadCaseState()
-    }, [caseId])
+        loadCaseState(id, token)
+    }, [id])
```

Comment:
should also depend on token and isAuthenticated. Check if token is not null before loadingCaseState